src/Parser/Body/ShippingLabelsParser.php

```diff
@@ -20,8 +20,10 @@ protected function extractShippingLabelInformationsFrom($order): ShippingLabel
         $instance = new ShippingLabel();
 
         $value = $this->extractValueFromXmlElement($order, 'RiferimentoNumeroLinea', false);
-        foreach ($value as $v) {
-            $instance->addLineNumberReference((string) $v);
+        if(!is_null($value)) {
```

Comment:
Grazie, se riesci a cambiare il check in `$value !== null` così è in linea con il resto della libreria